src/resources/views/crud/columns/select.blade.php

```diff
@@ -1,11 +1,36 @@
 {{-- single relationships (1-1, 1-n) --}}
+@php
+    $attributes = $crud->getRelatedEntriesAttributes($entry, $column['entity'], $column['attribute']);
+
+    if(count($attributes)) {
+        $lastKey = array_key_last($attributes);
+    }
+    $column['escaped'] = $column['escaped'] ?? true;
+    if(!empty($column['wrapper'])) {
+        $column['wrapper']['element'] = $column['wrapper']['element'] ?? 'a';
+    }
```

Comment:
It's also needed in `wrapper_end.blade.php` because everything you defined in `wrapper_start.blade.php` would not be available in `wrapper_end.blade.php`.